### PR TITLE
docs: add podman remote tutorial

### LINKED ISSUE #8280: Document podman remote features
### Is your enhancement related to a problem? Please describe

We should document podman remote features as well as the caveat that this will not work with public RSA keys, but rather use ed25519.

### Describe the solution you'd like

Docs on podman remote features

### Describe alternatives you've considered

_No response_

### Additional context

Reference: https://github.com/containers/podman-desktop/pull/8265

### PR DESCRIPTION
**Pull request by cdrage**
docs: add podman remote tutorial

### What does this PR do?

* Adds a podman remote tutorial to the website so you can learn how to
  access remote systems. This is taken from PR descriptions /
  explanations as well as the podman remote tutorial doc: https://github.com/containers/podman/blob/main/docs/tutorials/remote_client.md

### Screenshot / video of UI

N/A, see markdown

### What issues does this PR fix or reference?

Closes https://github.com/containers/podman-desktop/issues/8280

### How to test this PR?

N/A, docs

Signed-off-by: Charlie Drage <charlie@charliedrage.com>

### CODE REVIEW

File: website/docs/podman/podman-remote.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+Podman Desktop can manage remote Podman connections. This is facilitated through a list of connections using the command `podman system connection ls`.
+
+Containers can be created, started, stopped, and deleted as if managed locally.
+
+This functionality is enabled by connecting via SSH to the Podman socket on the remote host.
+
+**Ed25519** keys, an **SSH** connection, and an enabled **Podman Socket** are required for remote access.
+
+RSA keys are _not supported_; Ed25519 keys are the recommended and only current method to set up a remote connection.
```
Comment (axel7083) [inline website/docs/podman/podman-remote.md]: Do we have the upstream issue we could add as a footnote here ?

Comment (benoitf) [reply website/docs/podman/podman-remote.md]: https://github.com/mscdex/ssh2/issues/1375

Comment (axel7083) [reply website/docs/podman/podman-remote.md]: Thanks @benoitf

Comment (axel7083) [inline website/docs/podman/podman-remote.md]:
```suggestion
[RSA keys are not supported](https://github.com/mscdex/ssh2/issues/1375); Ed25519 keys are the recommended and only current method to set up a remote connection.
```

File: website/docs/podman/podman-remote.md [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+#### Prerequisites
+
+- SSH access to a Linux machine with Podman installed
+
+#### Procedure
+
+1. Enable Podman Remote in the Podman **Settings**:
+
+![Enable the remote setting](img/remote.png)
```
Comment (benoitf) [inline website/docs/podman/podman-remote.md:29]: wondering if we should have now both light and dark images so depending on the theme being used I'll see the correct image
https://docusaurus.io/docs/markdown-features/assets

File: website/docs/podman/podman-remote.md [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+
+2. Generate a local Ed25519 key:
```
Comment (benoitf) [inline website/docs/podman/podman-remote.md]:
```suggestion
2. Generate a local ed25519 key:
```

File: website/docs/podman/podman-remote.md [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+
+```sh
+$ ssh-keygen -t ed25519 -f ~/.ssh/id_ed25519
+```
+
+3. Copy your **public** Ed25519 key to the server:
```
Comment (benoitf) [inline website/docs/podman/podman-remote.md]:
```suggestion
3. Copy your **public** ed25519 key to the server:
```

File: website/docs/podman/podman-remote.md [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+```
+
+4. Enable the Podman socket on the remote connection:
+
+By default, the podman.socket is **disabled** in Podman installations. Enabling the systemd socket allows remote clients to control Podman.
+
+```sh
+$ systemctl enable podman.socket
+$ systemctl start podman.socket
+```
```
Comment (benoitf) [inline website/docs/podman/podman-remote.md]: maybe add a command to check if the socket or the service is there.

File: website/docs/podman/podman-remote.md [hunk 6]
```diff
@@ -0,0 +1,112 @@
...
+
+# Remote access
+
+Podman Desktop can manage remote Podman connections. This is facilitated through a list of connections using the command `podman system connection ls`.
+
+Containers can be created, started, stopped, and deleted as if managed locally.
+
+This functionality is enabled by connecting via SSH to the Podman socket on the remote host.
+
+**Ed25519** keys, an **SSH** connection, and an enabled **Podman Socket** are required for remote access.
```
Comment (benoitf) [inline website/docs/podman/podman-remote.md]:
```suggestion
**ed25519** keys, an **SSH** connection, and an enabled **Podman Socket** are required for remote access.
```

File: website/docs/podman/podman-remote.md [hunk 7]
```diff
@@ -0,0 +1,112 @@
...
+keywords: [podman desktop, containers, podman, remote]
+---
+
+# Remote access
+
+Podman Desktop can manage remote Podman connections. This is facilitated through a list of connections using the command `podman system connection ls`.
+
+Containers can be created, started, stopped, and deleted as if managed locally.
+
+This functionality is enabled by connecting via SSH to the Podman socket on the remote host.
```
Comment (benoitf) [inline website/docs/podman/podman-remote.md:15]: maybe add a section enlightening that it'll work only for `ssh connections of podman system connection` so if you're using a remote connection being a path to another directory, Podman Desktop won't connect to it.

**Review by benoitf — COMMENTED** [review]
To me, there is a blur line in the documentation.

From Podman Desktop side, I would expect a first part explaining how Podman Deskop will filter/connect and any restriction to connect to all `podman system connection`


and a different section on how to enable remote access if I have a VM/machine using podman.

Because here it seems we're more advertising how to do the setup of Podman which is more a 'podman feature'

**Comment by cdrage** [comment]
> To me, there is a blur line in the documentation.
> 
> From Podman Desktop side, I would expect a first part explaining how Podman Deskop will filter/connect and any restriction to connect to all `podman system connection`
> 
> and a different section on how to enable remote access if I have a VM/machine using podman.
> 
> Because here it seems we're more advertising how to do the setup of Podman which is more a 'podman feature'

Agreed, good if I instead re-order and have basically the first part of the procedure being "Enable Remote connections and you'll see any remote connections propagate on Podman Desktop GUI, etc."? 

The rest can be the "podman" setup.

**Comment by benoitf** [comment]
@cdrage yes

File: website/docs/podman/podman-remote.md (hunk 7 above)
Comment (cdrage) [reply website/docs/podman/podman-remote.md:15]: Next sentence says that ed key, ssh connection and an enabled podman socket is required, so it implies folder / other directory wont work, only SSH.

**Comment by cdrage** [comment]
> @cdrage yes

Updated 👍

**Review by benoitf — APPROVED** [review]

**Pull request merged**